### PR TITLE
Refactoring UI for readability

### PR DESCRIPTION
**Pull request by EdwardAndress**
When I was playing around with the layout I got into a right mess.  Hopefully these tweaks will make future attempts to change the layout easier.  Let me know what you think.

### CODE REVIEW

File: server.R [hunk 1]
```diff
@@ -319,8 +319,7 @@ function(input, output, session) {
   })
   
   addPopover(session, "topic_documents", "Questions in the topic",
-             content = paste0("This table contains all of the information on the questions asked on this topic.<br>",
-                              "You can choose which columns to show/hide by clicking on the \"Column Visibility\" button."),
```
Comment (EdwardAndress) [inline server.R]: We no longer have the Column Visibility button

File: ui.R [hunk 2]
```diff
@@ -1,115 +1,146 @@
 source(file = "global.R")
 ############# UI
 
+## Conditions
+searchTextEntered = "input.question.length > 0"
+tableHasRows      = "document.getElementsByClassName('odd').length > 1"
+topicChosen       = "input.topic_choice.length > 0"
+
 navbarPage("PQ Text Analysis",
-           theme = shinytheme("spacelab"),
```
Comment (EdwardAndress) [inline ui.R]: Changes to indenting

File: ui.R [hunk 3]
```diff
@@ -1,115 +1,146 @@
...
-                              min = min(data$Date),
-                              max = max(data$Date),
-                              start = min(data$Date),
-                              end = max(data$Date)
-               ),
-               bsTooltip("q_date_range", "Choose the time period you wish to search.",
-                         "auto", options = list(container = "body"))
-             )
+        conditionalPanel(
+          condition = searchTextEntered,
```
Comment (EdwardAndress) [inline ui.R]: abstracted logic

File: server.R (hunk 1 above)
Comment (RossKen) [inline server.R]: @EdwardAndress I have changed this in my PR

**Comment by RossKen** [comment]
My indenting was a bit of a mess when adding/taking things away so good to fix it.

**Pull request merged**